packages/atomic/src/components/atomic-result-template/atomic-result-template.tsx

```diff
@@ -6,6 +6,11 @@ import {
 } from '@coveo/headless';
 import {MapProp} from '../../utils/props-utils';
 
+/**
+ * The `atomic-result-template` component can is used to determine the format of query results depending on
+ * defined conditions. A `template` element must be a child of an `atomic-result-template` and an `atomic-result-list` must be
+ * the parent of each `atomic-result-template`.
```

Comment:
🤔  is it necessary to add new lines, won't it affect the way its rendered in the docs?

Reply:
You make a very good point. It does generate newline escapes, but doesn't cause issues in my generation scripts. That said, readability in comments shouldn't come at the cost of future issues. I'll adjust this and other entries. Thanks!